### PR TITLE
Add repository presets and interface to pick them

### LINKED ISSUE #125: Make it easier to only enable a few extensions
Is it possible to have a button to invert the currently selected extensions/skins to be enabled, or to set them all to disabled, rather than needing to click each manually? (Though it should probably require a skin be enabled so that it works)

### PR DESCRIPTION
**Pull request by MatmaRex**
Fixes #125

### CODE REVIEW

**Comment by edg2s** [comment]
I like this but I think we need to make it easier to discover which extensions are included or excluded from certain lists. For example if I am a designer on the Growth team, it would be pretty difficult for me to work out that GrowthExperiments is not included in "Wikimedia".

**Comment by edg2s** [comment]
One possibility would be to maintain a separate control to make the list of extensions visible, instead of only showing them when 'Other' is selected. If the user modifiers any of the checkboxes, 'Other' is automatically selected.

**Comment by edg2s** [comment]
We should also notify the teams/people using PD whose extensions will no longer included in the "Wikimedia" default:
* FlaggedRevs (cc @DannyS712)
* GlobalWatchlist (cc @DannyS712)
* GrowthExperiments (cc @catrope)
* ReplaceText (included in the initial commit, why is this in the list?)
* UploadWizard (that was me)
* WikiLambda (cc @jdforrester)

File: index.php [hunk 1]
```diff
@@ -86,15 +88,50 @@
...
+									'data' => 'tarball',
+									'label' => new OOUI\HtmlSnippet( '<abbr title="Skins and extensions included in the official MediaWiki release">Tarball</abbr>' ),
+								],
+								[
+									'data' => 'none',
+									'label' => 'None',
+								],
+								[
+									'data' => 'other',
+									'label' => 'Other',
```
Comment (edg2s) [inline index.php:114]: Should this be "Custom"?

**Comment by MatmaRex** [comment]
Tweaked the interface like you suggested.

**Comment by MatmaRex** [comment]
Before | After
------------ | -------------
image | image
image | image

**Comment by edg2s** [comment]
@MatmaRex Do you know why ReplaceText is included?

**Comment by MatmaRex** [comment]
Because it's included in the release tarball (as of 1.35).

**Pull request merged**